protos/CMakeLists.txt

```diff
@@ -23,44 +23,62 @@ add_library(srf_architect_protos)
 
 add_library(${PROJECT_NAME}::srf_architect_protos ALIAS srf_architect_protos)
 
+add_library(srf_test_protos)
+
 protobuf_generate_grpc_cpp(srf_protos
   HDRS srf_protos-headers
   PROTOS
-    srf/protos/codable.proto
-    srf/protos/remote_descriptor.proto
-    srf/protos/tensor_meta_data.proto
+  srf/protos/codable.proto
+  srf/protos/remote_descriptor.proto
+  srf/protos/tensor_meta_data.proto
 )
 
 protobuf_generate_grpc_cpp(srf_architect_protos
   GEN_GRPC
   HDRS srf_architect_protos-headers
   PROTOS
-    srf/protos/architect.proto
+  srf/protos/architect.proto
+)
+
+protobuf_generate_grpc_cpp(srf_test_protos
+  GEN_GRPC
+  HDRS srf_test_protos-headers
+  PROTOS
+  srf/protos/test.proto
 )
 
 # Create a custom target just for the generated headers which is necessary for style checks without build
 add_custom_target(
   srf_protos-headers-target
-  DEPENDS ${srf_protos-headers} ${srf_architect_protos-headers}
+  DEPENDS ${srf_protos-headers} ${srf_architect_protos-headers} ${srf_test_protos-headers}
 )
 
 # Disable clang-tidy and IWYU for the proto target
 set_target_properties(
   srf_protos
-    PROPERTIES
-      CXX_CLANG_TIDY ""
-      C_INCLUDE_WHAT_YOU_USE ""
-      CXX_INCLUDE_WHAT_YOU_USE ""
-      EXPORT_COMPILE_COMMANDS OFF
+  PROPERTIES
+  CXX_CLANG_TIDY ""
+  C_INCLUDE_WHAT_YOU_USE ""
+  CXX_INCLUDE_WHAT_YOU_USE ""
+  EXPORT_COMPILE_COMMANDS OFF
```

Comment:
All of the indentation here got messed up.

Reply:
Where is that formatting triggered? Am I missing an extension?

Reply:
We will sort out the format settings in another PR